db/migrate/20160113184103_create_posts_tags_join_table.rb

```diff
@@ -0,0 +1,8 @@
+class CreatePostsTagsJoinTable < ActiveRecord::Migration
+  def change
+    create_join_table :posts, :tags do |t|
+      # t.index [:post_id, :tag_id]
+      # t.index [:tag_id, :post_id]
```

Comment:
Nie wykomentowałem, tak się utworzyło automatycznie. W pliku schema.rb indeksy są.
  ----- Original Message ----- 
  From: Anna Ślimak 
  To: kraryk/test_blog 
  Cc: kraryk 
  Sent: Friday, January 15, 2016 10:22 AM
  Subject: Re: [test_blog] BŁąd przy wywoływaniu @post.tags (#3)


  In db/migrate/20160113184103_create_posts_tags_join_table.rb:

> @@ -0,0 +1,8 @@
> +class CreatePostsTagsJoinTable < ActiveRecord::Migration
> +  def change
> +    create_join_table :posts, :tags do |t|
> +      # t.index [:post_id, :tag_id]
> +      # t.index [:tag_id, :post_id]
czemu to wykomentowales? indexy sa wazne

  —
  Reply to this email directly or view it on GitHub.